chapters/publication.tex

```diff
@@ -99,91 +181,265 @@ \section{Collaborating on academic writing}
-allows them to evaluate the credibility of your work
-as well as to re-use your materials for further research.\sidenote{\url{https://dimewiki.worldbank.org/wiki/Exporting_Analysis}}
-If you have followed the steps in this book carefully,
-you will find that this is a very easy requirement to fulfill.\sidenote{
-\url{https://www.bitss.org/2016/05/23/out-of-the-file-drawer-tips-on-prepping-data-for-publication/}}
-You will already have a publishable (either de-identified or constructed)
-version of your research dataset.
-You will already have your analysis code well-ordered,
-and you won't have any junk lying around for cleanup.
-You will have written your code so that others can read it,
-and you will have documentation for all your work,
-as well as a well-structured directory containing the code and data.
+\section{Preparing a complete replication package}
+
+While we have focused so far on the preparation of written materials for publication,
+it is increasingly important for you to consider how you will publish
+the data and code you used for your research as well.
+More and more major journals are requiring that publications
+provide direct links to both the code and data used to create the results,
+and some even require being able to reproduce the results themselves
+before they will approve a paper for publication.\sidenote{
+  \url{https://www.aeaweb.org/journals/policies/data-code/}}
+If your material has been well-structured throughout the analytical process,
+this will only require a small amount of extra work;
+if not, paring it down to the ``replication package'' may take some time.
+A complete replication package should accomplish several core functions.
+It must provide the exact data and code that is used for a paper,
+all necessary de-identified data for the analysis,
+and all code necessary for the analysis.
+The code should exactly reproduce the raw outputs you have used for the paper,
+and should include no documentation or PII data you would not share publicly.
+
+\subsection{Publishing data for replication}
+
+Enabling permanent access to the data used in your study
+is an important contribution you can make along with the publication of results.
+It allows other researchers to validate the mechanical construction of your results,
+to investigate what other results might be obtained from the same population,
+and test alternative approaches to other questions.
+Therefore you should make clear in your study
+where and how data are stored, and how and under what circumstances it might be accessed.
+You do not always have to complete the data publication data yourself,
+as long as you cite or otherwise directly reference data that you cannot release.
+Even if you think your raw data is owned by someone else,
+in many cases you will have the right to release
+at least some subset of your analytical dataset or the indicators you constructed.
+Check with the data supplier or other professional about licensing questions,
```

Comment:
Add here the importance of establishing a data licensing agreement if directly acquiring data from partners (e.g. administrative data that is not otherwise public, or proprietary data from firms, etc).

Reply:
Could this go in data collection? or even sooner where we talk about consent etc?

Reply:
created issue #351